xformers/ops.py

```diff
@@ -655,6 +655,46 @@ def from_arguments(
         )
 
 
+class Chunk3(torch.autograd.Function):
```

Comment:
Can we make a dispatcher function that calls into `Chunk3.apply`, and which is what users use?